rpm/pext.spec

```diff
@@ -0,0 +1,44 @@
+#Disable debug packages
+%define debug_package %{nil}
+
+Name:	pext
+Version:	0.9
+Release:	1%{?dist}
+Summary:	Python-based extendable tool
+
+License:	GPLv3
```

Comment:
Shouldn't this be GPLv3+?

Reply:
I based that on the LICENSE file in the repository but I can update it.

Reply:
It's a GPL oddity. There's no way to state "or any later version" in the license file, only in the headers of your code files: https://github.com/Pext/Pext/blob/master/pext/pext.py#L9-L10. I wish that could be clearer as well, but hey, that's the GPL.